cobalt/browser/metrics/cobalt_metrics_service_client.cc

```diff
@@ -41,66 +38,41 @@ class CobaltMetricsLogUploader : public metrics::MetricsLogUploader {
                  const std::string& log_hash,
                  const std::string& log_signature,
                  const metrics::ReportingInfo& reporting_info) {
-    // TODO(b/372559349): Add logic here to report the Blob to the WebApp.
-    NOTIMPLEMENTED();
+    DLOG(INFO) << "UMA Payload uploading! Hash: " << log_hash;
```

Comment:
Can we leave NOTIMPLEMENTED here until it is?

Reply:
Done